### ISSUE TITLE
BCELoss pos_weight parameter

### ISSUE THREAD

**Issue by Jokeyrs** [pytorch/pytorch#67007]
Why **BCEWithLogitsLoss** has pos_weight and **BCELoss** doesn't?
And why they are different classes, doesn't it better to make one class for BCE loss with parameter **from_logits**=True/False?

cc @albanD @mruberry @jbschlosser @walterddr
